lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -21,9 +21,49 @@
             <layout.heading local-class='Heading'>
                 <div local-class='Title'>{{t 'registries.drafts.draft.form.new_registration'}}</div>
             </layout.heading>
-            <layout.left local-class='Left'>
-                {{!-- Sidenav: page headings, validations --}}
-            </layout.left>
+            
+            <layout.leftNav as |leftNav|>
+                {{#each draftManager.pageManagers as |pageManager index|}}
+                    <PageLinkManager
```

Comment:
I am in favor of simply naming this <PageLink> - as it does little bit of work from passed in state to figure out the css class (for display purposes). We could pass `leftNav.link` and style it there. That way we consolidate all the magic in one place.

Reply:
@fabmiz So are you proposing naming the component differently?

Reply:
So *don't* use the manager pattern here?

Reply:
@adlius Right. This is a display component. Doesn't load any data or have any actions, just computed props to derive css classes.